controllers/todoAppController.js

```diff
@@ -190,21 +190,36 @@ angular.module('toDoApp.controllers', ['toDoApp.services','auth.services'])
 
 
     }])
-    .controller('loginControl',['$scope','$state','$window','AuthenticationService',function($scope,$state,$window,authService){
+    .controller('loginControl', ['$scope', '$state', '$window', 'AuthenticationService', function ($scope, $state, $window, authService) {
 
-        if(authService.CheckSession()){
+        if (authService.checkSession()) {
             $state.go('home');
         }
-        $scope.login = function(user){
-        authService.Login(user.email,user.password)
-            .then(function(response){
-                $window.sessionStorage.setItem('token',response);
-                $state.go('home');
-            })
-            .catch(function(error){
-                $scope.loginError = true;
-                console.log(error);
-            })
-    };
-
-}]);
\ No newline at end of file
+        $scope.login = function (user) {
+            authService.Login(user.email, user.password)
+                .then(function (response) {
+                    $window.sessionStorage.setItem('token', response);
+                    $state.go('home');
+                })
+                .catch(function (error) {
+                    $scope.loginError = true;
+                    console.log(error);
+                })
+        };
+
+    }])
+    .controller('registrationControl', ['$scope', '$state', 'AuthenticationService', function ($scope, $state, authService) {
+        $scope.register = function (user, form) {
+            if (form.$valid) {
+                authService.CreateUser(user)
+                    .then(function (response) {
+                        console.log(response.data);
```

Comment:
what we do after the registration is complete? do we show a message? or something